src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5380,8 +5383,8 @@ export class StudyViewPageStore {
             ) {
                 const yAxisBinCount = MutationCountVsCnaYBinsMin;
                 const xAxisBinCount = 50;
-                const bins = (await internalClient.fetchClinicalDataDensityPlotUsingPOST(
```

Comment:
[Better formatting for inline await expression nested in calls](https://prettier.io/blog/2019/11/09/1.19.0.html#better-formatting-for-inline-await-expression-nested-in-calls-6856-by-thorn0)